Library/Formula/weechat.rb

```diff
@@ -14,6 +14,10 @@ class Weechat < Formula
 
   option "with-perl", "Build the perl module"
   option "with-ruby", "Build the ruby module"
+  option "with-python", "Build the python module"
```

Comment:
It looks like by default support for those scripts were not here, for every language (which surprised me quite a bit, as stated in first message, as on Linux it's compiled with all of them by default).
That's why I added explanation for all of them to allow to compile with them (see line 38 to 44 asking to remove language support if option do not ask for this language).

By the way, what about building with them by default? Maybe can bottle build have them by default, as Linux packages do have it?